### PR TITLE
Fix __setattr__

### PR DESCRIPTION
**Pull request by jjcasmar**
If attribute is not a data or a link, calls the default implementation for __setattr__. This allows to have properties for custom types defined by a user.

### CODE REVIEW

**Review by jnbrunet — APPROVED** [review]
Good catch @jjcasmar , thank you for this improvement. Let's just wait for the CI before merging.

**Comment by jjcasmar** [comment]
Should we merge this fix also in master? Its not there :-s

**Pull request merged**
